wiki/How_You_Can_Help!/en.md

```diff
@@ -1,80 +1,78 @@
 # How you can help!
 
-_This is a post that [peppy](/users/2 "peppy") made some time ago and requested to make it an article here. [You can see the original post here](/community/forums/topics/83155)._
+osu! itself has a large, and diverse community from all walks of life, and anyone can help make osu! more awesome for you, and for everyone.
 
-I've been wanting to make a post about this for a while, but have been holding off while I get things in place on the site and in my head. As you are probably aware, the osu! community and ecosystem is growing at a constant rate, but the team hasn't grown in size at all. I'm sure with a community this big, we have a lot of hidden talent out there, so the idea of this post is to reach out and tell you all that we are always looking for help and input, as well as outlining the current areas where you can immediately jump in and help out!
+## Help out with [the wiki](https://github.com/ppy/osu-wiki)
 
-Firstly, from a personal perspective I am already completely saturated. I spend all my free time on osu! (or osu!stream) and do not have any more resources I can throw at it. This means that most of the stuff that gets done on osu! is based on my personal priorities. This doesn't mean that it has to be a limiting factor — there are so many opportunities to make osu! more awesome without needing the access or abilities of a developer.
+The wiki is our official knowledge base about anything and everything about osu! The wiki is looking for more contributors to make the site up to date with fresh new knowledge, and to reach a wider audience via translations.
 
-Here is a current list of areas I can think of where anyone can help out, ordered by the most important in my eyes. I'll try and keep this updated as I see things come up, but don't hesitate to make suggestions (preferably in this thread) if you have any.
+Read more about how you can get started contributing to the wiki [here!](https://osu.ppy.sh/help/wiki/osu!_wiki_Contribution_Guide)
```

Comment:
You can shorten this link to `/wiki/osu!_wiki_Contribution_Guide` (I think this is the preferred form for wiki links)